lib/reporters/dot_reporter.js

```diff
@@ -43,8 +46,10 @@ module.exports = class DotReporter {
       this.currentLineChars = 0;
       this.out.write('\n  ');
     }
-    if (result.passed) {
+    if (result.passed && !result.todo) {
```

Comment:
Help jog my memory.. is a `todo` reported as pass as well? If not, probably wouldn't need `!result.todo` ? (goes towards the rest fo the changes in the PR)

Reply:
For QUnit a todo can either pass or fail. A pass means all assertions passed, even though it was marked as todo. QUnit's UI treats this as a failure, even though internally it's marked as passing, because you should stop marking it as a todo if it passes.

Reply:
Thanks for the summary!